docs/advanced/model_examples.rst

```diff
@@ -11,11 +11,14 @@ The examples below make use of a pandas dataframe ``df`` containing dummy data,
     from plothist import get_dummy_data
     df = get_dummy_data()
 
+.. note::
+   It is not recommended to copy and paste code from this page to reproduce the shown examples.
```

Comment:
The note is a good idea. However, is it really not recommended? I would rather strongly advertise the fact that the full scripts used for the examples are in the gallery. In its current form, the note advise the user to skip the documentation to go directly to the gallery, but the documentation still has some useful tips and comments for the new users, that the gallery doesn't show.

Reply:
recommended -> necessary + add note